README.md

```diff
@@ -28,6 +22,18 @@ we recommend installing `casperlabs-client` in an isolated Python 3 environment
 or
 [virtualenv](https://virtualenv.pypa.io/en/latest/).
 
+These steps should help you set up your Python environment for the client.
+
+```
+sudo apt install gcc
+sudo apt install python3.7
+sudo apt-get install python3.7-dev
+virtualenv -p python3.7 env
```

Comment:
This will fail for Ubuntu 20.04.  There is no python3.7 package.  If there is a python3.8 package on 18.04, it would probably be safest to use that instead.

Reply:
I believe that 18.04 was only up to 3.7.  So we might need to sections for 18.04 and 20.04.

Reply:
Looks like I can't suggest an edit via web with the "```" which suggest uses.

I would have the above as 18.04 instructions.

Then have 20.04 instructions with:

sudo apt install gcc
sudo apt install python3.8
sudo apt-get install python3.8-dev
virtualenv -p python3.8 env